apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/VedtaksbrevRoute.kt

```diff
@@ -63,5 +64,26 @@ fun Route.vedtaksbrevRoute(service: VedtaksbrevService, behandlingKlient: Behand
                 }
             }
         }
+
+        get("vedtak/payload/reset") {
+            withBehandlingId(behandlingKlient) {
+                val brevId = requireNotNull(call.parameters["brevId"]).toLong()
+                val sakId = requireNotNull(call.parameters["sakId"]).toLong()
```

Comment:
Denne har vi ein mekanisme for å hente ut automatisk (i `RouteUtils`), så du kan berre bruke sakId som variabel direkte utan å definere her. Men det ser ikkje ut som verken den eller brevId er i params, så da vil det nok ikkje funke.

Foreslår å anten leggje inn sakid og brevid i stien, eller å ta dei inn som body.